sdk/messaging_servicebus/tests/service_bus.rs

```diff
@@ -0,0 +1,101 @@
+#![cfg(all(test, feature = "test_e2e"))] // to run this, do: `cargo test --features test_e2e`
+extern crate log;
+
+use azure_messaging_servicebus::service_bus::Client;
+use chrono::Duration;
+
+#[tokio::test]
+async fn send_message_test() {
+    let mut client = create_client().unwrap();
+    client
+        .send_message("hello, world!")
+        .await
+        .expect("Failed to send message");
+}
+
+#[tokio::test]
+async fn receive_and_delete_message_test() {
```

Comment:
Does this expect `send_message_test` to run first? it seems like we want these tests to be one test instead of many.

Reply:
That's how I had done it initially, I'll revert back to that ~ done ✅